src/config/config.js

```diff
@@ -21,6 +21,9 @@ const Config = {
   DEFAULT_GAS_PRICE: 0.0000004,
   CREATE_DORACLE_GAS_LIMIT: 1500000,
   UNLOCK_SECONDS: 86400,
+  deployment: {
+    TESTNET_ONLY: false,
```

Comment:
LGTM, but can be more convenient to set this from a cmd flag

Reply:
done 👍